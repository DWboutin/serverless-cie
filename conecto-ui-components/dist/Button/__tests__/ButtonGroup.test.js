"use strict";

var _interopRequireWildcard = require("@babel/runtime/helpers/interopRequireWildcard");

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _enzyme = require("enzyme");

var _Button = _interopRequireWildcard(require("../../Button"));

global.console = {
  error: jest.fn()
};
describe('ButtonGroup', () => {
  it('should match snapshot', () => {
    const wrapper = (0, _enzyme.shallow)(_react.default.createElement(_Button.ButtonGroup, null, _react.default.createElement(_Button.default, null, "Test1"), _react.default.createElement(_Button.default, null, "Test2")));
    expect(wrapper).toMatchSnapshot();
  });
  it('should have the prefix class', () => {
    const wrapper = (0, _enzyme.shallow)(_react.default.createElement(_Button.ButtonGroup, null, _react.default.createElement(_Button.default, null, "Test1"), _react.default.createElement(_Button.default, null, "Test2")));
    expect(wrapper.instance().classesPrefix).toEqual('to-btn-group');
    expect(wrapper.hasClass(wrapper.instance().classesPrefix)).toEqual(true);
  });
  it('should console.error when not passing children', () => {
    (0, _enzyme.shallow)(_react.default.createElement(_Button.ButtonGroup, null));
    expect(global.console.error).toBeCalled();
  });
});