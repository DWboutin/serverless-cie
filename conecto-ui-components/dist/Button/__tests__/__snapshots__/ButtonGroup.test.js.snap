// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup should match snapshot 1`] = `
<div
  className="to-btn-group"
>
  <Button
    comp="button"
    htmlType="button"
    type="primary"
  >
    Test1
  </Button>
  <Button
    comp="button"
    htmlType="button"
    type="primary"
  >
    Test2
  </Button>
</div>
`;
